components/monitoring/logging/base/external-secrets/monitoring-logging-splunk-forwarder.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: external-secrets.io/v1beta1
+kind: ExternalSecret
+metadata:
+  name: monitoring-logging-splunk-forwarder
+  annotations:
+    argocd.argoproj.io/sync-options: SkipDryRunOnMissingResource=true
+    argocd.argoproj.io/sync-wave: "-1"
+spec:
+  dataFrom:
+  - extract:
+      key: "" # will be added by the overlays
```

Comment:
so we effectively make the secret invalid without an extra patch - will cause some code to fail somewhere...